homeassistant/components/homematicip_cloud/light.py

```diff
@@ -56,6 +57,9 @@ async def async_setup_entry(
         elif isinstance(device, AsyncWiredDimmer3):
             for channel in range(1, 4):
                 entities.append(HomematicipMultiDimmer(hap, device, channel=channel))
+        elif isinstance(device, AsyncDinRailDimmer3):
```

Comment:
Since the outcome here is effectively the same as `AsyncWiredDimmer3`, does it makes sense to combine these branches? E.g.:

```python
elif isinstance(device, (AsyncWiredDimmer3, AsyncDinRailDimmer3)):
```

Reply:
👍 You're totaly right